.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: true
```

Comment:
enabling blank issues is something we have to avoid most of the time, there are already issue templates